Doc/whatsnew/3.12.rst

```diff
@@ -861,6 +861,16 @@ New Features
   get a frame variable by its name.
   (Contributed by Victor Stinner in :gh:`91248`.)
 
+* Add :c:func:`PyErr_GetRaisedException` and :c:func:`PyErr_SetRaisedException`
+  for saving and restoring the current exception.
+  These functions take and return a single exception,
```

Comment:
```suggestion
* Add :c:func:`PyErr_GetRaisedException` and :c:func:`PyErr_SetRaisedException`
  for saving and restoring the current exception.
  These functions return and accept a single exception object,
```

* Clarify that we're talking about a single exception object (or maybe another word is more appropriate for the C API?), as the current text implies that it previously accepted/returned multiple _exceptions_ rather a tuple of multiple objects that comprised one exception
* The current text implies both functions both accept and return an exception
* The functions are listed out of order to the verbs that respectively apply to them.
* Could use more idiomatic/conventional language (accept vs. take)

Reply:
That's a nice improvement; thanks!